app/src/main/java/com/example/taskwise_eventmaster/MainActivity.kt

```diff
@@ -90,7 +92,7 @@ class MainActivity : ComponentActivity() {
                                         )
                                     }
 
-                                    navController.navigate("profile")
+                                    navController.navigate("home_page")
```

Comment:
Minor nitpick :D `page` is used mainly in a Web context, here you can use `screen`. ex. `home_screen`